DeploymentCreatedProcessorTest.shouldCloseSubscriptionWhenInCorrectOrder is flaky

```java
2019-04-25T13:46:21.641Z] [ERROR] shouldCloseSubscriptionWhenInCorrectOrder(io.zeebe.broker.workflow.processor.DeploymentCreatedProcessorTest)  Time elapsed: 10.886 s  <<< FAILURE!

[2019-04-25T13:46:21.641Z] org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>

[2019-04-25T13:46:21.641Z] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)

[2019-04-25T13:46:21.641Z] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)

[2019-04-25T13:46:21.641Z] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)

[2019-04-25T13:46:21.641Z] 	at io.zeebe.test.util.TestUtil$Invocation.until(TestUtil.java:96)

[2019-04-25T13:46:21.641Z] 	at io.zeebe.test.util.TestUtil$Invocation.until(TestUtil.java:69)

[2019-04-25T13:46:21.641Z] 	at io.zeebe.test.util.TestUtil.waitUntil(TestUtil.java:41)

[2019-04-25T13:46:21.641Z] 	at io.zeebe.broker.workflow.processor.DeploymentCreatedProcessorTest.shouldCloseSubscriptionWhenInCorrectOrder(DeploymentCreatedProcessorTest.java:126)

[2019-04-25T13:46:21.641Z] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

[2019-04-25T13:46:21.641Z] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

[2019-04-25T13:46:21.641Z] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

[2019-04-25T13:46:21.641Z] 	at java.lang.reflect.Method.invoke(Method.java:498)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)

[2019-04-25T13:46:21.641Z] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)

[2019-04-25T13:46:21.641Z] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)

[2019-04-25T13:46:21.641Z] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)

[2019-04-25T13:46:21.641Z] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)

[2019-04-25T13:46:21.641Z] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)

[2019-04-25T13:46:21.641Z] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)

[2019-04-25T13:46:21.641Z] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)

[2019-04-25T13:46:21.641Z] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)

[2019-04-25T13:46:21.641Z] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)

[2019-04-25T13:46:21.641Z] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.Suite.runChild(Suite.java:128)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.Suite.runChild(Suite.java:27)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)

[2019-04-25T13:46:21.641Z] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)

[2019-04-25T13:46:21.642Z] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)

[2019-04-25T13:46:21.642Z] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)

[2019-04-25T13:46:21.642Z] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeLazy(JUnitCoreWrapper.java:119)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:87)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:158)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)

[2019-04-25T13:46:21.642Z] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
```

--- a/broker-core/src/test/java/io/zeebe/broker/util/LogStreamPrinter.java
+++ b/broker-core/src/test/java/io/zeebe/broker/util/LogStreamPrinter.java
@@ -114,6 +114,8 @@ public class LogStreamPrinter {
   private static void writeRecordHeader(final LoggedEvent event, final StringBuilder sb) {
     sb.append("Position: ");
     sb.append(event.getPosition());
+    sb.append(" Key: ");
+    sb.append(event.getKey());
   }
 
   private static void writeWorkflowInstanceBody(

--- a/broker-core/src/test/java/io/zeebe/broker/workflow/processor/DeploymentCreatedProcessorTest.java
+++ b/broker-core/src/test/java/io/zeebe/broker/workflow/processor/DeploymentCreatedProcessorTest.java
@@ -120,6 +120,8 @@ public class DeploymentCreatedProcessorTest {
 
     // when
     writeMessageStartRecord(3, 1);
+    waitUntil(
+        () -> rule.events().onlyDeploymentRecords().withIntent(DeploymentIntent.CREATED).exists());
     writeNoneStartRecord(7, 2);
 
     // then

